MetaMask/metamask-extension
# UX: Release the AvatarGroup for Account Menu List Items

## **Description**

This PR removes the feature flag for showing asset icons in the account list menu.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open the account list menu
2. See an avatar for an empty account
3. See a series of avatars for an account with multiple tokens, sorted by most total value of token.

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="398" alt="SCR-20231213-juwe" src="https://github.com/MetaMask/metamask-extension/assets/46655/c4d1c92f-4fdf-4e2b-b37a-2cf752a04cb6">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
